combat-trainer.lic

```diff
@@ -1425,6 +1520,9 @@ class SpellProcess
         data['cast'] = "cast #{moon}"
       end
     end
+    
+    game_state.casting_regalia = true if data['abbrev'] == 'REGAL'
```

Comment:
💡 Could this be `game_state.casting_regalia = data['abbrev'] == 'REGAL'`?

Reply:
Apparently it can be!